internal/billing/repository_test.go

```diff
@@ -74,8 +74,16 @@ func TestRepository_MonthlyActiveUsers(t *testing.T) {
 	TestGenerateActiveUsers(t, conn)
 
 	today := time.Now().UTC()
-	threeMonthsAgo := time.Date(today.AddDate(0, -3, 0).Year(), today.AddDate(0, -3, 0).Month(), 1, 0, 0, 0, 0, time.UTC)
-	oneMonthAgo := time.Date(today.AddDate(0, -1, 0).Year(), today.AddDate(0, -1, 0).Month(), 1, 0, 0, 0, 0, time.UTC)
+	// Some time calculations are impacted when using the current day vs. the
+	// start of the month. For example, if...
+	// today -> May 30th
+	// today.AddDate(0, -3, 0).Month() -> March
+	// February was expcted here, but we get March. This seems to be a
+	// rounding thing since February 30th is not a valid date. Instead, the
+	// start of the month is used to ensure the correct months are calculated.
+	monthStart := time.Date(today.Year(), today.Month(), 1, 0, 0, 0, 0, time.UTC)
```

Comment:
Not that we risk having any months with fewer than 1 days anytime soon, but why make this relative at all? Why not just
```suggestion
	monthStart := time.Date(2000, time.May, 1, 0, 0, 0, 0, time.UTC)
```
etc?

Reply:
There would need to be a lot more modifications to the test. `TestGenerateActiveUsers` generates data for the last twelve months.